snowfakery/fakedata/fake_data_generator.py

```diff
@@ -26,32 +26,41 @@ def RealisticMaybeRealEmail(self):
 
 
 class FakeData:
-    """Wrapper Faker which adds Salesforce names and case insensitivity."""
+    """Wrapper for Faker which adds Salesforce names and case insensitivity."""
 
     def __init__(self, faker: Faker):
-        self.faker = faker
-        self.fake_names = FakeNames(faker)
+        faker = faker
+        fake_names = FakeNames(faker)
 
-        # canonical form of names is lower-case
-        self._lowers = {
-            v.lower(): v for v in dir(self.fake_names) if not v.startswith("_")
+        def obj_to_func_list(obj: object, canonicalizer: T.Callable):
+            return {
+                canonicalizer(name): getattr(obj, name)
+                for name in dir(obj)
+                if not name.startswith("_") and name != "seed"
```

Comment:
What is "seed"?

Reply:
Is it clearer in 975d5f2?

Reply:
Not really. I knew we were ignoring "seed" but not why. Now it doesn't mention "seed" at all and I'm not sure if that's an oversight in the refactoring or what.

Reply:
Okay trying to clarify again: fc91d7e